includes/user.php

```diff
@@ -447,7 +447,7 @@ function ap_user_top_posts_tab(){
     
     $link = '?tab=';
     ?>
-    <?php printf(__('Top %s', 'ap'), $active); ?>
+    <?php printf(__('Top %s', 'ap'), __(ucfirst($active), 'ap')); ?>
```

Comment:
@rahularyan I know it's not a perfekt solution, but it works to translate this part. Without that you can't translate the "Top question" and "Top answer" part under activity in the user profile.
* /user/[username]/activity/
* /user/[username]/activity/?tab=answers
* /user/[username]/activity/?tab=questions

If `$active` is empty, it is set to "answers". Otherwise `$active` will be contain sanitized string from the `$_GET['tab']` param (answers or questions). And because there is no "questions"-string (in lowercase) in the translation file, i convert the first character to uppercase.